### DISCUSSION TITLE
Compare signatures

### DISCUSSION THREAD

**Discussion by ertankucukoglu** [lsh123/xmlsec#733] [Q&A]
Hello,

I am completely new to this  XMLDSig and pretty much confused.

I am using v1.3.2 (openssl) release of xmlsec

I have two signed XML files provided. I am not signing them.
Both individual files signatures verify OK.
But I know second one with double signature does not match the first one.
What I am told is (one to one translation)
"signature value of the first file does not match the second file approved certificate."

I wonder if it is possible to use xmlsec for figuring out if two such files check.

```
PS D:\xmlsec> .\xmlsec.exe --version
xmlsec 1.3.2 (openssl)
PS D:\xmlsec> .\xmlsec.exe --verify .\GIB-9370646431-202304-KB-000000.xml
Verification status: OK
PS D:\xmlsec> .\xmlsec.exe --verify .\9370646431-202304-Y-000000.xml
Verification status: OK
PS D:\xmlsec>
```

Thanks & Regards,
Ertan

**Comment by lsh123** [comment]
I am not sure I understand what is the problem and w/o files it's impossible to even try to answer.

**Comment by ertankucukoglu** [comment]
Hi, Thanks for the reply.
How can I share files in private?

**Reply by lsh123** [reply]
Sorry this is an open source project, I don't do private consulting. You can try to reproduce the problem with non-proprietary files you can share.

**Reply by ertankucukoglu** [reply]
I'll see if I can find such files. Thanks.
